### PR TITLE
add chatlist specific scopes

### PR DESCRIPTION
**Pull request by seekdavidlee**
create a separate chatlist specific scope

### CODE REVIEW

File: packages/mgt-chat/src/statefulClient/chatOperationScopes.tests.ts [hunk 1]
```diff
@@ -20,3 +20,24 @@ describe('chatOperationScopes tests', () => {
...
   });
 });
+
+describe('chatListOperationScopes tests', () => {
+  it('should have a minimal permission set', () => {
+    const expectedScopes = [
+      'User.Read',
+      'User.ReadBasic.All',
+      'User.Read.All',
+      'People.Read',
```
Comment (plasne) [inline packages/mgt-chat/src/statefulClient/chatOperationScopes.tests.ts:30]: Are we sure some of these aren't duplicate permissions?

Like People.Read.All doesn't encompass People.Read? Or User.Read.All doesn't encompass User.Read and User.ReadBasic.All?

Comment (seekdavidlee) [reply packages/mgt-chat/src/statefulClient/chatOperationScopes.tests.ts:30]: I think they are likely to encompass, I am following mgt's convention.

Comment (salujamanish) [reply packages/mgt-chat/src/statefulClient/chatOperationScopes.tests.ts:30]: yes, User.Read.All will encompass User.Read and User.ReadBasic.All

Comment (plasne) [reply packages/mgt-chat/src/statefulClient/chatOperationScopes.tests.ts:30]: So they request everything? that seems odd.

That doesn't seem in the spirit of "minimal", but maybe ask for clarity from Sebastian what the intent is.

**Review by plasne — APPROVED** [review]

**Review by kavithasreenath — APPROVED** [review]

**Pull request merged**